Makefile

```diff
@@ -349,7 +349,8 @@ endif
 .PHONY: build-windows
 build-windows:
 	@echo "===> Building Antrea bins and antrea/antrea-windows Docker image <==="
-	docker build --cache-from antrea/base-windows:$(WIN_BUILD_TAG) -t antrea/base-windows:$(WIN_BUILD_TAG) -f build/images/base-windows/Dockerfile --network $(DOCKER_NETWORK) $(WIN_BUILD_ARGS) .
+	$(eval NO_EXIST=$(shell docker pull antrea/base-windows:$(WIN_BUILD_TAG); echo $$?))
+	@if [ "$(NO_EXIST)" = 1 ]; then echo "Build antrea/base-windows:$(WIN_BUILD_TAG)"; docker build --cache-from antrea/base-windows:$(WIN_BUILD_TAG) -t antrea/base-windows:$(WIN_BUILD_TAG) -f build/images/base-windows/Dockerfile --network $(DOCKER_NETWORK) $(WIN_BUILD_ARGS) .; else echo "Skip building existing base-windows:$(WIN_BUILD_TAG) image"; fi
```

Comment:
I feel like we should just assume it already exists in this target, just like we do for `antrea-ubuntu`?

Reply:
Yes, it's more reasonable to unify implementation of these two targets. I will add this logic to another place.